src/vs/workbench/contrib/terminal/electron-browser/terminalInstanceService.ts

```diff
@@ -117,9 +117,14 @@ export class TerminalInstanceService extends Disposable implements ITerminalInst
 		return this._instantiationService.createInstance(LocalPty, id);
 	}
 
-	public async attachToProcess(id: number): Promise<ITerminalChildProcess> {
-		await this._localPtyService.attachToProcess(id);
-		return this._instantiationService.createInstance(LocalPty, id);
+	public async attachToProcess(id: number): Promise<ITerminalChildProcess | undefined> {
+		try {
```

Comment:
same thing as the above. don't need it to fix this issue and may not need at all

Reply:
Let's keep